src/azure-cli/azure/cli/command_modules/batchai/tests/latest/test_batchai_custom.py

```diff
@@ -945,7 +945,7 @@ def test_list_setup_task_files_for_cluster_when_output_on_bfs(self, get_files_fr
                 relative_mount_path='bfs',
                 container_name='container',
                 credentials=None),
-            os.path.join('path/segment/', '.'), 60)
+            os.path.join('path', 'segment', '.'), 60)
```

Comment:
What's the difference by adding `/.`

Reply:
@qwordy The old one only works on Linux-like systems. Any hardcoded slashes or anti-slashes are not recommended.

Reply:
@qwordy I think there's no difference between `{path}` and `{path}/.`, but in order to keep the original code I decided to not change it.

Reply:
already fixed in this PR: https://github.com/Azure/azure-cli/pull/14239/files#diff-a7bbbe4eef8bb85685e4c7965541d92d